index.js

```diff
@@ -1,10 +1,10 @@
 import React, { PureComponent } from 'react';
-import { View, StyleSheet, ART, Text } from 'react-native';
+import { View, StyleSheet, Text } from 'react-native';
 import PropTypes from 'prop-types';
 
 import barcodes from 'jsbarcode/src/barcodes';
 
-const { Surface, Shape } = ART;
+import {Surface, Shape} from '@react-native-community/art';
```

Comment:
@jomla97 Should we also update the package.json?

Reply:
Good call @wonsikin , done!